src/resmom/mom_hook_func.c

```diff
@@ -3159,6 +3197,9 @@ record_job_last_hook_executed(unsigned int hook_event,
 			*p = '\0';
 			p_dir = filepath;
 		}
+	} else {
+		log_event(PBSEVENT_ERROR, PBS_EVENTCLASS_HOOK,
+			  LOG_ERR, __func__, "Hook not received");
 	}
```

Comment:
Please use log_event/log_eventf here: because for Windows it will log wrong error code
https://github.com/PBSPro/pbspro/blob/f84fe1585683331d08711c94ff02a11087b80f50/src/resmom/mom_hook_func.c#L3265
https://github.com/PBSPro/pbspro/blob/f84fe1585683331d08711c94ff02a11087b80f50/src/resmom/mom_hook_func.c#L3270
https://github.com/PBSPro/pbspro/blob/f84fe1585683331d08711c94ff02a11087b80f50/src/resmom/mom_hook_func.c#L3275
https://github.com/PBSPro/pbspro/blob/f84fe1585683331d08711c94ff02a11087b80f50/src/resmom/mom_hook_func.c#L3286